drivers/drivers.go

```diff
@@ -1,5 +1,12 @@
 package drivers
 
+const (
+	// KdmpConfigmapName kdmp config map name
+	KdmpConfigmapName = "kdmp-config"
+	// KdmpConfigmapNamespace kdmp config map ns
+	KdmpConfigmapNamespace = "kube-system"
```

Comment:
It is better to defaultAdminNamespace value as it is used during configmap creation.